keyboards/system76/launch_1/rules.mk

```diff
@@ -1,33 +1,23 @@
-# MCU name
-MCU = atmega32u4
+# AVR MCU options
+MCU = atmega32u4# MCU name
+BOOTLOADER = atmel-dfu# Bootloader selection
+F_CPU = 8000000# CPU frequency divided by two since AVR is at 3.3 V
+F_USB = 16000000# External oscillator is 16 MHz
```

Comment:
```suggestion
# MCU name
MCU = atmega32u4

# CPU frequency divided by two since AVR is at 3.3 V
F_CPU = 8000000

# External oscillator is 16 MHz
F_USB = 16000000

# Bootloader selection
BOOTLOADER = atmel-dfu
```

Reply:
Done, again, hesitantly, because there is a good reason for doing it the other way. Basically, it enables tracking down the places where inbuilt assumptions of no trailing whitespaces are located for these special variables. So that if a user inadvertently adds one it will not break a build like it does today. Do not assume we all use GNU Make.

Reply:
I think it's actually a fairly reasonable assumption. For instance, [the CLI prefers `gmake` before `make`](https://github.com/qmk/qmk_firmware/blob/master/lib/python/qmk/commands.py#L22) when compiling boards, and on macOS, the Homebrew formula [depends on GNU `make`](https://formulae.brew.sh/formula/make). The FreeBSD install script also specifies `gmake`.